### DISCUSSION TITLE
[온보딩 미션] 가이드 (& 멤버들 자기소개) [과거자료]

### DISCUSSION THREAD

**Discussion by statsops** [Pseudo-Lab/Anxious-Lookout#11] [Announcements]
# 온보딩 미션

<초조한 전망대>에 참여하기 위해서는 아래 미션을 진행하셔야 가입하실 수 있습니다. 

가짜연을 비롯한 커뮤니티 활동의 큰 걸림돌은 동료들의 불참입니다. 
맡은 일을 해 오지 않거나, 그냥 조용히 사라지는 사람들이 매 번 있습니다. 
이런 분들이 많아질 수록 같이 활동하시는 분들의 의욕을 해치고 공동체의 사기가 떨어지게 됩니다. 

따라서 **적극적이고 존중하는 태도, 성실하게 참여하는 의지, 뚜렷한 관심사,** 그리고 **소통 능력**이 가장 중요한 덕목입니다.   
이 미션을 마침으로써 참가자 여러분은 본인이 그런 사람이라는 걸 입증하실 수 있게 됩니다. 
기존 멤버분들은 여러분들을 챙겨드리고 싶은 마음이 가득합니다! 이 활동들을 통해 다가와 주시길 바랍니다. 

## 미션 가이드! 
상호작용을 촉진하기 위해서 가이드는 살짝 불친절하게 설계되어 있습니다. 
주체적으로 관심을 표현하시고, 적극적으로 질문하고, 불편한 부분 개선에 기여해 주세요. 

온보딩 미션은 각자 시간에 개인별로 진행하시면 됩니다! 토요일에 얼마나 진행되었는지 같이 이야기할 수 있는 시간을 갖습니다.
저희는 봉사정신으로 모인 사람들입니다! 그러므로 주저 말고 채팅방에서 물어봐 주세요 

### 1. 이 글에 댓글을 남겨주세요
이 미션을 마친 분들께 오픈채팅방 공유 주소가 전달됩니다. 
아래의 미션을 마치고 대댓글의 링크를 [가입 대기 카톡](https://open.kakao.com/o/s9284d9g) 에 남겨주세요.

양식은 다음과 같습니다. 
제목: '본인 이름' [온보딩 미션] 
내용: 간단한 자기소개를 작성해 주세요. 
- AI의 어떤 부분에 관심이 있는지, 어떤 걸 하고 싶은지 등이 들어가면 좋습니다. 
- AI 관련 경험이나 교육이 있으면 밝혀 주시면 좋습니다. 

### 2. 정보 탐색의 첫 시작하기
- [써클 아카이브](https://github.com/Pseudo-Lab/impatient-observatory/discussions/4) 탭 글에 정보를 탐색할 수 있는 각종 소스를 정리해두었습니다.
- 이곳을 천천히 살펴보고, 관심 있는 웹을 찾아 본인 댓글의 대댓글로 공유해주세요!
- 또한, 본인이 이미 즐겨 보고 있는 웹이 있다면 [써클 아카이브](https://github.com/Pseudo-Lab/impatient-observatory/discussions/4) 탭에 댓글로 작성해주세요!

### 3. 초조한 전망대 첫 기여하기
- [정보 모음](https://github.com/Pseudo-Lab/impatient-observatory/discussions/25) 탭 글에 해당 주차 글을 찾아 작성한 후, 본인 댓글의 대댓글에 링크를 남겨주세요. 
- 'm월 n일' 댓글의 대댓글에 글을 작성하시면 됩니다. 위 링크로 들어가시면 바로 이해하실 수 있을거에요! 
#### 예시
- 이번 주에 찾은 좋은 뉴스를 남겨주세요.  
- 좋은 정보 채널 (유튜브 채널 / 카톡 채널/  블로그 / 커뮤니티 등등) 을 남겨주세요
- 양질의 AI 교육 자료를 남겨주세요. 
- 좋은 행사가 있다면 알려주세요 
- 기타 'ai에 관심 있는 사람이 좋아할 법한 자료'를 남겨주세요!  

### 4. 관심있는 스터디 찾아보기
- 과거의 아카데믹 프로젝트를 찾아보고 “들었으면 좋았겠다” 하는거 찾은 후, 미션지 대댓글에 남겨주세요 
- 8기 스터디까지는 노션에 정리가 되어있고, 9기 스터디부터는 가짜연구소 깃허브 레포로 확인할 수 있습니다. 
- 시간이 조금 걸리더라도 관심 분야를 찾거나, 이후 스터디 빌드를 위해 참고할 수 있는 부분이기 때문에 반드시 진행해주시길 바랍니다!
- [가짜연구소 노션 페이지](https://chanrankim.notion.site/c9013228f63342b689a96e18c0db32c8?pvs=4)<img width="1175" alt="스크린샷 2024-06-25 오전 8 50 08" src="https://github.com/Pseudo-Lab/impatient-observatory/assets/35090655/4ffa99d5-ed04-4cbf-8fc4-4db0c5b98c10">
- [가짜연구소 깃허브 페이지](https://github.com/orgs/Pseudo-Lab/repositories)

### 5. 내가 뭘 기여할 수 있을까? 나의 성장 플랜(+ 가짜연구소에서 뭘 하고싶은지)을 참여자 스스로 정의해보기 
저(한결)의 경우를 예시로 남겨볼게요! 
- 저는 웹 개발자였고, NLP를 사용해서 프로그램을 개발하는 사람이 되고 싶었습니다. 당시에는 모두가 모델러를 해야 된다는게 중론이었다보니 미적통계선대를 배워야 하는 줄 알고 프리랜서로 일을 하며 차근히 준비를 하기로 마음 먹었죠. 
- 저의 최초 성장 플랜은 방송대에서 학위를 두개를 따서 학문의 기초를 쌓고, NLP는 기초부터 현재까지 반년에 하나 꼴로 책을 마치는 식으로 접근을 했었습니다. 우선은 개발 포지션으로 ML 프로젝트에 참여하면서 천천히 넘어가려고 했고요. 
- 가짜연구소를 비롯한 커뮤니티를 활동을 통해, 이 성장 플랜이 확실히 부스트 되었습니다. 저는 바로 아카데믹 빌더부터 시작했어요. 저는 사람 모아서 뭔가 하는걸 좋아하는 편이고,  '사람들을 모아서 가르치자! 그러면 내가 제일 많이 성장한다'는걸 예전부터 알고 있었거든요. 
왜냐면 이렇게 하면 매 주 데드라인을 뒀다 보니 꼭 얼마씩은 공부를 해 가야 했어요. 제가 진행하다 보니 확실하게 알아야 되서 더 그랬고요. 
- 무엇보다 큰 소득은 현업에 계신 선배님들을 만날 수 있었던 점입니다. '요즘 모델링 안해도 되고, 뽑는 일자리 대부분이 데이터 엔지니어쪽이다'라는 걸 알게 되었거든요. 그분들 덕에 내가 쓸 데 없는 짓 하고 있다는걸 파악할 수 있었고, 그 이후의 성장 플랜까지 잡아주셨어요. 결론적으로 커뮤니티 덕을 저는 많이 본거죠. 
- 현재의 제 역할은 가짜연구소에서 받은 만큼 돌려주기 위해 '모임을 더 키우고 결과물을 더 퍼트리는 역할'을 하려고 해요. 우선 LLM과 관련된 최신 정보들을 큐레이션해서 웹에 올리고, 그걸 유통시키는 작업을 제가 만든 스터디인 <깃헙에 NLP 잔디심기 시즌 5> 에서 진행하려고 합니다. 많은 사람들이 모여서 기여하고, 또 그 모인 사람들 속에서 저를 발전시켜줄 수 있는 사람을 찾는 겁니다. 또 <초조한 전망대>를 운영하면서  좋은 사람들을 가짜연구소로 더 받아들이고, 정보 탐색도 같이 진행하고 싶어요. 

- 제 역할 정의는 오래 한 만큼 구체적입니다 ㅎㅎ. 여러분들은 '역할 정의' 쪽에서는 이렇게까지 쓸 내용이 없을 거에요. 러너 대부분은 '나보다 경험이 많은, 혹은 배움에 열정이 있는 사람들을 만나고, 이 분들과 즐겁게 같이 공부하면서 성장하는 것' 정도가 될 겁니다. 하지만 어떻게 성장하고 싶은지에 대한 부분은 꼭 알아야 합니다. 그래야 뭘 배울지 정하기라도 하겠죠? 

- 그런 부분을 서술해 봅시다 

### 6. 오프라인 참여
- 첫 오프라인 모임에는 가능한 반드시 참석하셔서 초조한 전망대에서 활동하시는 분들을 만나시길 바랍니다. 
- 매 월 마지막 주 토요일, 장소는 매번 온라인에서 얘기하여 정하고 있습니다. 
- 참가 신청은 오픈카톡방에서 토요일 이전에 투표를 통해 진행하고 있습니다. 
- 오프라인으로 만나 방탈출, 한강 피크닉 등을 진행하여 서로 친해지고 있으니 가능하면 필히 참여하셨으면 좋겠습니다! 

---
## 하면 좋은 미션!!
- 초조한 전망대에 참여하는 많은 분들은 다른 누군가와 대화하거나, 해커톤을 나가는 등 혼자서 하기 어려운 것들을 위해 들어오셨을 것이라 생각합니다. 
- 이러한 만남에서 중요한 것은 두 가지라고 생각합니다. 첫 번째는 자발적이고 능동적인 참여이고, 두 번째는 그 자취를 남기는 것입니다. 
- 가짜연구소의 많은 분들은 이미 자신이 관심 있는 분야를 공부하기 위해 스터디를 빌드하거나 러너로 참여하며 링크드인과 블로그 등을 통해 자신이 하고 있는 일들을 자세히 남기고 있습니다. 
- 여러분들도 스터디를 청강 혹은 참여해보고 최대한 많은 것들을 스스로 남겨보는 습관을 길러봅시다! 

### 1. 내게 맞을 것 같은 아카데미 프로그램 **청강**해보고 미션지 댓글에 후기 남기기
- ⚠️ 종료된 스터디가 많을 수 있습니다! 대부분의 스터디는 시즌제로 국내 학기와 비슷한 시기에 진행되기 때문에 해당 기간에 진행중인 스터디가 없다면 추후에 진행해주세요!! 
- 가짜연구소 Discord 주소: https://discord.gg/QAMzXhM7 

### 2. 친해지고 싶은 멤버 찾아보기 & 그분들에게 커피챗 DM을 보내고, 만남 후기 댓글에 남기기
- AI, 데이터분석, 개발이라는 분야에서 특이한 문화 중 하나는 커피챗인것 같습니다. 관심은 있으나 잘 모르는 누군가와 만나 이야기하고 인사이트를 나누는 일이죠. 여러분도 가짜연구소 내에서 많은 분들과의 커피챗을 통해 인사이트를 얻어보아요!
- <초조한 전망대> 안에서 찾아보셔도 됩니다. 많은 분들이 커피챗을 거절하지는 않을 것입니다. 시간 관계 상 오프라인으로 얘기하는 것이 어렵다면, 디스코드에서 온라인으로 얘기하는 것도 가능하니 편하게 물어봐주세요!
- 종종 디스코드 DM을 안 보시는 경우가 많습니다! DM에 무응답인 경우, 운영진에게 얘기하거나 링크드인 등을 활용해보세요!! 

### 3. 개인 블로그, SNS를 글을 작성하고 공유하기
- 대부분의 경우, 단순히 무언가를 듣고 읽는 것으로는 충분하지 않을 수 있습니다. 특히, 취업이나 대학원 진학 등을 위해서는 더더욱 그럴 것입니다. 자신을 알리고 브랜딩하는 것이 중요해진 시기에 우리는 어떤 것들에 관심이 있고 어떤 것을 해왔는지 발자취를 남길 필요가 있습니다.
- 링크드인, 블로그 등 원하는 플랫폼을 하나 선택해서 만들어 봅시다. 
- 만든 플랫폼에 자신이 공부하고 있는 것들을 자신의 언어로 표현해봅시다. 어디선가 떠온 느낌이 드는 글보다는, 오래걸리더라도 자신의 언어로 충분히 이해한 글을 쓰는 것이 중요합니다.
- 좋은 예시가 있으면 편할 것 같아 가짜연구소의 찬란님 [링크드인](https://www.linkedin.com/in/chanran-kim/)을 남겨드립니다 

### 4. 운영 참가하기
- 초조한 전망대에 있다보면 월 말에 같이 할 것들을 찾아보거나 같이 먹을 곳 들을 찾아보는 등 운영에 도움을 요청드릴 수도 있습니다.  
- 이런 것들을 하면서 같이 친해지길 바랍니다!!

### 5. 간단한 모임을 기획하고 파일럿 만들어보기
- 재미있고 유익한 활동을 한번 기획해 봅니다.  
    - 작은 규모로, 3~5주 정도의 스터디를 해 볼 수 있습니다  
    - 워크샵을 열어볼 수 있을 것 같아요 
- 같이 하실 분들을 한번 모아보세요! 

🥇 이제 댓글을 남기고 미션을 시작해 봅시다!

**Comment by JeongSuJin** [comment]
온보딩 미션 1번에 대한 답입니다!
들었으면 좋았겠다 하는 프로젝트는 8기의 랭체인소맨 입니다.
https://chanrankim.notion.site/a7632025ae0646f7b53539e4d81d17a2

**Reply by HangryDev** [reply]
좋네요! 다음 기수에도 비슷한 주제의 스터디를 진행하시는지 DM으로 물어봐도 좋겠습니다.

**Comment by yunha-p** [comment]
사용성 연구를 했어서 그런지, 올해 상반기에 진행됐던 **[인과추론팀] 온라인 통제 실험 연구자로 거듭나기** 의 아카이빙 자료를 흥미롭게 봤습니다. 저는 오프라인 실험만 진행해봤었고, 최근 머신러닝 공부를 시작해서 제가 갖고 있던 지식들에 대한 새로운 인사이트를 얻을 수 있어서 좋았습니다👍 열심히 공부해서 언젠간 스터디에 참여해보고 싶네요..!

**Reply by yunha-p** [reply]
디코 자유게시판이 너무 조용하길래 일단 깃헙에 코멘트로 남깁니다🥹

**Reply by HangryDev** [reply]
좋아요! 토요일날 뵙는게 기대됩니다-

**Comment by statsops** [comment]
안녕하세요? 가짜연구소 DSF(Data Science Fellowship) 러너로 활동중인 강호현입니다!

현재 클라우드 엔지니어로 활동하고 있습니다.

graphRAG에 관심이 많습니다. 최근에 쿠버네티스를 열심히 공부하고 있어요.

데이터 엔지니어링 공부와 병행해서, Kubernetes 기반 LLM/AI 서빙을 목표로 하고 있습니다.

**Comment by seyongk** [comment]
안녕하세요! 저는 JAX 스터디 러너로 활동 중인 김세용입니다.

저는 AI 개발자로 활동하고 있고요, 최근에는 embodied ai, agent ai 연구를 하고 있습니다.

그 외에는 사이드 프로젝트로 게임 개발에 관심이 있습니다. 

게임 개발 대박나서 플렉스 하는 것을 목표로 하고 있습니다 ^^

**Reply by statsops** [reply]
JAX는 어떤 느낌인가요? 게임 개발자로 성장하실 생각이신가요. 화이팅입니다.

**Comment by hidden719** [comment]
함경표
- 인공지능학과 석사과정을 마치고, AI의 지속적인 흐름을 따라가고자 가짜연에 가입했습니다.
- 현재는 구직중에 있구요. 최근에는 LLM Agent 공부를 했습니다.
- 현재 목표는 데이터 사이언티스트+ 데이터 엔지니어로 가는 것 입니다.

**Reply by statsops** [reply]
멋진 LLM Agent가 기대되네요!

**Comment by Yushin-L** [comment]
안녕하세요! 저는 병원의 여러 DB를 구성하고 연구 환경을 제공하는 회사에서 근무중인 이유신입니다!

현재는 의료에 관련된 언어 데이터 및 모델을 활용할 방법에 대해 연구하고 있습니다.

그 외에 개발 관련된 방향으로 많은 사람들이 편하게 사용할 수 있도록 도커와 같은 가상 환경 툴을 공부하려 하고 있습니다.

**Reply by statsops** [reply]
병원의 DB.. 보안이 많이 중요한 부분이군요. 도커 툴 공부도 화이팅입니다!

**Comment by YunSeok-Kang** [comment]
안녕하세요!  저는 미디어 산업의 제작 단가를 낮추는 기술에 관심 있는 강윤석입니다. 이를 통해 더욱 많은 사람이 개인의 이야기를 쉽게 풀어나갈 수 있는 세상을 만드는 것을 꿈꾸고 있습니다.

컴퓨터공학을 전공했으며, 본교 대학원에 진학하여 3D Vision 연구를 진행하고자 준비하고 있습니다. 가짜연구소에서는 3D-GS/NeRF 논문 읽기 모임 8기 러너로 참여 중입니다.

미디어가 점점 개인화 되어가는 세상에서 음악과 동영상은 꼭 대중적이지 않아도 독특함으로 승부를 볼 수 있는 세상이 되었으며,  대규모 프로덕션이 만든 게임 못지않은 인디 게임의 인기를 보면 이는 거스를 수 없는 시대의 흐름으로 생각됩니다.

하지만 여전히 다른 미디어에 비해서 게임의 제작 비용 감소 폭은 적다고 생각합니다. 저는 기술을 통해 이 문제를 풀어가보고 싶습니다. 다양한 생각을 가진 사람들이 적은 위험 부담으로 자신만의 독특한 경험을 게임으로 만들어낼 수 있다면, 그것이 우리의 오늘날을 기록하는 더욱 생생한 방법으로 자리매김 할 수 있지 않을까요?

위의 방향성을 가지고 학계에 발을 들인 후, 최종적으로는 저의 제품을 만들어내는 S/W 엔지니어로 살아가고 싶습니다. 특히, 마음 맞는 사람들끼리 모여 다시 한번 우리를 대표하는 게임을 만들어보고 싶네요. 

개인 홈페이지: https://www.yskang.kr/about.html

**Reply by statsops** [reply]
와.. 단가 최적화 멋진걸요! 윤석님의 제품이 궁금하네요!

**Reply by YunSeok-Kang** [reply]
사실 제 게임 만들 때 돈 아껴주는 제품에 관심이 많습니다 ^0^

**Comment by howsmyanimeprofilepicture** [comment]
안녕하세요. AI 개발자 이홍규입니다. AI로 현실의 문제를 푸는 것에 관심이 많습니다. 많이 배우고 있습니다. :)

**Reply by statsops** [reply]
AI 아티스트 홍님, 멋진 AI 개발자 되시길 기대합니다.

**Reply by howsmyanimeprofilepicture** [reply]
TMI: 홍대병, 아티스트병, 엔지니어병이 심합니다...

**Comment by yesinkim** [comment]
안녕하세요. NLP Engineer로 김예신입니다. 호기심이 많아서 여기 저기 기웃거리다가 여기까지 흘러오게 되었습니다. 

최근에는 인공지능의 활용성에 대해서 깊이 고민하고 있습니다. k8s, LLM, 모델 경량화등에 관심을 가지고 있습니다만 체력이슈로 힘들어하고 있는 중입니다. 😭

**Reply by statsops** [reply]
체력.. 중요하군요. 멋지게 롱런해봐요~

**Comment by gdone9009** [comment]
안녕하세요. 자기소개를 하자면
마법학교에 입학한 강동원이에요:) 
모두모두 잘 부탁드려요~

경영학 전공으로 비전공자이지만 데이터를 다루는 공부를 하고자 열심히 공부하게 되었습니다 

가짜연구소에서 지식과 경험을 공유하면서 역량을 향상시키고, 협업과 토론을 통해 다양한 관점에서 문제를 바라보고 해결해가는 것을 기대하고 있습니다.

최근에 프롬프트 엔지니어링, 이미지 사운드 제작이나 sLLM 파인튜닝 분야에 관심을 갖게 되었는데, 같은 관심사를 갖고 계신 분이 있으면 좋겠어요ㅎㅎ

http://bit.ly/마법학교song

그리고, 매우 최근에 방송대 통계데이터과학과에 편입학하였는데요. 아직 잘 모르지만, 3학년 편입 1학기인데요. 수강신청은 2학년과목인 빅데이터의이해와활용 3학년과목은 데이터처리와활용 4학년과목인 마케팅조사 도전하고 있습니다.

복수전공은 미디어방송학과이며, 재미있는거 위주로 디지털영상편집 영화기획제작 미디어혁신스토리텔링 수강신청했습니다.

전체적으로 결님의 쪼렙버전이라고 보시면 됩니다. 느린학습자가 열심히 배우고 따라가겠습니다.
성실하게 참여하겠습니다.

**Reply by statsops** [reply]
느리지만 꾸준하게! 멋지게 성장하시길 기원합니다.

**Reply by HangryDev** [reply]
환영합니다! 동원님은 정말 행사 강의 많이 다니셔서 ㅎㅎ 이벤트나 정보 잘 알려주셔서 넘 좋습니다. 앞으로도 많은 기여 부탁드립니다`~

**Comment by Garurubi** [comment]
안녕하세요. AI 엔지니어 정재희입니다.
Spring 국비 교육을 받고 취업해서 어쩌다 보니 현재는 오픈소스 sLLM을 활용해서 일을 하고 있습니다. AI 기술의 변화가 신기하고 공부하는게 재미있어서 여기까지 오게 되었습니다. 잘 부탁드립니다.

**Comment by Heejung-cho** [comment]
안녕하세요! 👋
디자이너 조희정이라고 합니다 :)
다양한 AI 프로그램을 사용하여 디자인에 접목하고 싶어서 관심을 갖게 되었어요.
최근 부스트코스에서 모두를 위한 AI활용 강의를 들었습니다.
잘부탁드려요 😄

**Comment by yukyeongmin** [comment]
안녕하세요. AI researcher를 희망하고 있는 유경민이라고 합니다.
현재는 diffusion팀 러너로 참여하고 있는데 추가적으로 듣는다면 multimodal도 들어보고 싶어요.

요즘에는 어쩌다보니 이미지나 영상 diffusion 보다는 3D쪽을 더 보고 있어요.
초조한 전망대에서 전반적인 소식도 듣고 열심히 공부하기 위한 에너지를 얻을 수 있으리라 생각해 참여하고 싶습니다.

**Comment by seojw9292** [comment]
안녕하세요.
마법학교에 입학한 경제학 전공자 서중원입니다. 비전공자이지만 기술과 인공지능에 큰 관심을 가지고 학습하고 있습니다. 최근에는 RAG를 공부하고 있으습니다. 

깃잔심 스터디와 초조한 전망대에서 지식과 경험을 공유하며 역량을 키우고, 협업과 토론에 참여할 수 있으면 좋겠습니다

아직 부족한 점이 많지만, 꾸준히 학습하고 참여하며 성장하겠습니다. AI 관련 기사와 뉴스를 꾸준히 팔로우하며 커뮤니티 내에서 의미 있는 기여를 할 수 있기를 희망합니다. 잘 부탁드립니다 :)

**Comment by spyhop23** [comment]
안녕하세요. 김지연입니다.
현재 데이터사이언스 석사과정 중입니다.
학부 전공이 이 쪽이 아니다보니 공부를 하는 중에도 한계가 있는 것 같아
좋은 분들과 다양한 지식과 경험을 함께 나누고 배우고자 참여 신청을 하게 되었습니다.
특히 컴퓨터비전(CV) 쪽에 관심을 갖고 있어 추후 해당 분야의 논문 스터디를 희망하고 있고
함께 대회도 나가고 싶은 마음 또한 있습니다.
잘 부탁드립니다 :)

**Comment by Eunbin-Jung** [comment]
안녕하세요! 현재 UX와 HCI에 관심이 많아 해당 전공으로 대학원을 다니고 있는 UX 디자이너 정은빈이라고 합니다 :)
다양한 AI 지식을 배우고 해당 경험을 통해 디자인에 활용하고 싶어 신청하게 되었습니다.
가능하면 함께 협업을 통해 앱 개발이나 해커톤을 함께 진행하고 싶은 마음도 있습니다!
잘부탁드립니다 :)

**Comment by cognitive88** [comment]
안녕하세요. 컴퓨터구조 전공으로 석사과정 재학중인 전경호 입니다.

큰 주제로는 AI 가속인데 구체적으로 Processing-in-memory (PIM), Compute eXpress Link (CXL) 등 과 같은 Emerging Dram-based AI acceleration 을 연구중입니다. LLM, 3D vision 등의 AI workloads, GPU, TPU, vLLM 등과 같은 주제에도 관심 많습니다!

원래 AI 모델에도 관심이 많았는데, 대학원에 온 뒤로 컴퓨터 아키텍처 위주로 공부하고 연구를 하느라 AI 관련 다양한 지식 및 소프트웨어 능력이 많이 떨어졌습니다. 그래서 최근에 다시 핫한 모델들 및 JAX framework 를 공부하고 있습니다. 

가짜연구소 스터디 목록을 보면 제가 속한 시스템분야 관련 스터디는 거의 없는 것 같습니다. 저도 지식을 나누고 뭉치는걸 좋아해서 기회가 된다면 스터디를 열어보고 싶기도 해요. 그리고 저 또한 이곳에서 다양한 AI 및 소프트웨어 관련 지식을 배우고 싶습니다.

잘 부탁드립니다 :)

**Reply by Yushin-L** [reply]
안녕하세요! https://open.kakao.com/o/s9284d9g 로 대화주시면 감사하겠습니다 ㅎㅎ

**Comment by Dehkartes** [comment]
안녕하세요. 학사 출신  AI 엔지니어 허세진입니다.
주로 LLM 응용 비즈니스 로직 개발을 하며 NLP, 서비스 개발에 흥미있습니다.

AI관련 스터디나 캐글이나 데이콘같은 대회에 참여하고싶습니다.

감사합니다.

**Reply by Yushin-L** [reply]
안녕하세요!!

**Comment by 3hui112** [comment]
'김세희' [온보딩 미션]

안녕하세요. 인공지능 관련 석사과정을 준비하면서 인공지능에 대해 공부하고 있는 김세희입니다.
긍정적인 영향을 주고 받을 수 있는 커뮤니티를 운영관리해 주셔서 감사하다는 말씀을 드리고 싶습니다.
석사과정 입학을 준비하면서 부족한 점을 많이 느끼게 되어 도움을 받을 수 있는 커뮤니티에 속하고 싶다는 생각에 마법학교에 관심을 가지게 되었습니다.

NLP, CV, Multi-modal 등 도메인에도 구분 짓지 않고 다양한 분야에 관심을 가지면서 넓고 얕게 공부해 왔는데, 좀 더 심화내용을 공부해 보면서 제 적성에 더 적합한 분야와 도메인을 찾고 싶습니다. (궁극적으로는 도메인에 영향을 받지 않는 뛰어난 리서쳐가 되는 것이 꿈입니다. ㅎㅎ)

컴퓨터공학부에서 인공지능을 공부하면서 수학적 원리 뿐만 아니라 컴퓨터 인프라/하드웨어 등에도 관심이 많습니다. 실제로 클라우드 엔지니어를 꿈꾸기도 했고요. (정말 관심 있는 분야가 많습니다...) 마법학교를 통해 다양한 분들을 만나뵙고 교류할 수 있기를 희망합니다.

**Reply by 3hui112** [reply]
2. 써클 아카이브 탭 내에서 제가 관심있는 웹사이트 주소는 다음과 같습니다. 
https://huggingface.co/blog

**Reply by Yushin-L** [reply]
안녕하세요! [가입 대기 카톡](https://open.kakao.com/o/s9284d9g) 으로 연락주시면 감사하겠습니다!

**Reply by 3hui112** [reply]
3. 옴니파서2에 대한 소식을 전달했습니다.
https://github.com/Pseudo-Lab/impatient-observatory/discussions/25?sort=new#discussioncomment-12230828

**Comment by choi3num** [comment]
안녕하세요. 보안회사에서 일하고 있는 최명호라고 합니다.

보안 자동화에 관심이 있습니다.

NPL를 최소화한후 기존 보안회사들(안랩, 알약)등과 연동하여 보안 기능을 강화 하기를 원하고 있습니다.

처음이라 방향을 잡기 위해

1. n8bn 와 같은 Workflow를 만들고 있습니다.


2. 제가 가진 도메인 지식과 결합하여 커뮤니티에 기여하려고 합니다.
ex)
 - 내 보안은 내가 책임진다.(집에서의 보안 서버 구축-초소형)
 - 회사에서 사용하는 보안(PC & 서버)은 최소 내가 책임진다.

**Comment by Raymondgwangryeol** [comment]
'박나영' [온보딩 미션]
안녕하세요! CV에 대해 공부한 지 1년도 안 된 뉴비 박나영입니다.
Boostcamp AI 7기를 수료하고, 돌이켜 봤을 때 부족한 부분이 참 많았습니다.
특히 수학 기본기를 다져야겠다는 생각을 했고, 마침 가짜연구소에서 이와 관련된 스터디가 있기에 참여해 보고 싶습니다!

저는 현재 자율주행(자동차 OR 로봇)에 관심이 있어서, 현재 MOT부터 시작해서 C++, 임베디드 코딩 등을 배워 자율주행 회사에 입사하는 것이 목표입니다! 지금은 MOT를 다루는 회사를 몰색 중 입니다.

Kaggle로 구현 능력을 많이 기를 수 있을 것 같아서, kaggle 대회 참여에도 관심이 있습니다!
함께 정보를 나누고, 좋은 기운을 받아 성장할 수 있는 기회가 되었으면 좋겠습니다!

**Reply by Raymondgwangryeol** [reply]
써클 아카이브 탭 내에서 제가 관심있는 웹사이트는 다음과 같습니다!
[The MOST Important Skills To Learn in 2024 byTina Huang]
